core/src/test/scala/unit/kafka/coordinator/group/GroupMetadataManagerTest.scala

```diff
@@ -714,28 +714,6 @@ class GroupMetadataManagerTest {
     assertEquals(group, groupMetadataManager.addGroup(new GroupMetadata("foo", initialState = Empty)))
   }
 
-  @Test
-  def testDeleteEmptyGroup() {
-    val group = new GroupMetadata(groupId, initialState = Empty)
-    assertEquals(group, groupMetadataManager.addGroup(group))
-
-    EasyMock.expect(replicaManager.getMagic(EasyMock.anyObject())).andStubReturn(Some(RecordBatch.CURRENT_MAGIC_VALUE))
-    mockGetPartition()
-    EasyMock.replay(replicaManager, partition)
-
-    val result = groupMetadataManager.deleteGroups(Seq(group.groupId))
-    assert(result.size == 1 && result.contains(group.groupId) && result.get(group.groupId).contains(Errors.NONE))
-  }
-
-  @Test
-  def testDeleteNonEmptyGroup() {
```

Comment:
Why remove these test cases?

Reply:
They made a call to `GroupMetadataManager.deleteGroups(...)` that we just deleted. Similar tests exist in `GroupCoordinatorTest`.